src/server/game/Entities/Player/Player.cpp

```diff
@@ -7658,27 +7669,7 @@ void Player::UpdateArea(uint32 newArea)
     UpdateAreaDependentAuras(newArea);
 
     pvpInfo.IsInNoPvPArea = false;
-    if (!area)
-    {
-        RemoveByteFlag(UNIT_FIELD_BYTES_2, 1, UNIT_BYTE2_FLAG_SANCTUARY);
-        RemoveRestState();
-        return;
-    }
-
-    // Xinef: area should inherit zone flags
-    AreaTableEntry const* zone = sAreaTableStore.LookupEntry(area->zone);
-    uint32 areaFlags = area->flags;
-    bool isSanctuary = area->IsSanctuary();
-    bool isInn = area->IsInn(GetTeamId(true));
-    if (zone)
-    {
-        areaFlags |= zone->flags;
-        isSanctuary |= zone->IsSanctuary();
-        isInn |= zone->IsInn(GetTeamId(true));
-    }
-
-    // previously this was in UpdateZone (but after UpdateArea) so nothing will break
-    if (isSanctuary)    // in sanctuary
```

Comment:
can u eventually tell testers how to verify these things are still working somehow ingame after the rework?

Reply:
These checks are handled else where in the core, so they were removed.